web/operator/dist/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Same as above. It feels very fragile, if SNO's have to download a binary and a dist folder..